gfx/UltimateCataclysm/pngs_human_body_32x36/overlay/skin/skins_mutation.json

```diff
@@ -41,12 +41,12 @@
   },
   {
     "id": "overlay_female_mutation_THICK_SCALES",
-    "fg": "scales_f",
+    "fg": "scales_thick_f",
```

Comment:
Same here, name the sprite with the correct id

Reply:
I copied how it was done in the older sprites. Gonna do that if that's the new standard.